tpe/lib/src/Shape.cc

```diff
@@ -84,6 +84,12 @@ void BoxShape::SetSize(const math::Vector3d &_size)
   this->dirty = true;
 }
 
+//////////////////////////////////////////////////
+math::Vector3d BoxShape::GetSize()
```

Comment:
can you add a simple check [here](https://github.com/ignitionrobotics/ign-physics/blob/claire/tpelib_add_fns/tpe/lib/src/Shape_TEST.cc#L39) in Shape_TEST.cc to test this function?

Reply:
added https://github.com/ignitionrobotics/ign-physics/pull/45/files#diff-b6ac34fb28f6fcdad5799d1f97124481R40